_data/speakers.yml

```diff
@@ -19,3 +19,17 @@
   his efforts have been focused on data visualization tools, particularly the Altair and
   Vega-Lite libraries."
   photo: "vanderplas.jpg"
+- name: "Devi A S L"
+  title: "Independent Software Consultant"
+  introduction: |-
+    Devi is an independent software consultant, based in Bangalore.
+    She is a long term Pythonista and her work spans diverse domains ranging
+    from developing high-performance math libraries, cryptography to
+    architecting complex web applications.
+    
+    Through her work that surpassed over a decade, she has experienced diverse
+    work cultures at small startups, MNCs and remote teams where she has been
+    through the lows and highs women in tech from different perspectives.
+    She has been an active member of the PyCon India community since its
+    inception, as a volunteer and as a speaker
+  photo: "asldevi.jpg"
```

Comment:
Please use https://squoosh.app for optimising image payloads. After running the image through this, its size dropped from ~600kb to ~200kb.